plugin/constants.py

```diff
@@ -1,7 +1,9 @@
 import re
 
+DEFAULT_COMMAND = '${storage_path}/LSP-gopls/bin/gopls'
 PACKAGE_NAME = __package__.partition('.')[0]
 SESSION_NAME = 'gopls'
+SETTINGS = 'LSP-gopls.sublime-settings'
```

Comment:
`DEFAULT_COMMAND` is not used. Also slightly less repetition:

```suggestion
PACKAGE_NAME = __package__.partition('.')[0]
SESSION_NAME = 'gopls'
SETTINGS = '{}.sublime-settings'.format(PACKAGE_NAME)
```